test/auth.e2e-spec.ts

```diff
@@ -421,6 +421,86 @@ describe("AuthController e2e Tests", () => {
         });
     });
 
+    describe("Revoke Refresh Token DELETE /auth/refresh/userId", () => {
+        it("should return 200 if refresh token is revoked", async () => {
+            await loginAndGetTokens("l.castro@outlook.com", "password", app);
+
+            const { access_token, refresh_token } = await loginAndGetTokens(
+                "jessica.williamson@gmail.com",
+                "password",
+                app,
+            );
+
+            await prisma.user.update({
+                where: {
+                    email: "l.castro@outlook.com",
+                },
+                data: {
+                    refreshToken: null,
+                },
+            });
+
+            await request(app.getHttpServer())
+                .delete(revokeRTUrl)
+                .set("Cookie", [access_token, refresh_token])
+                .send({ email: "l.castro@outlook.com" })
+                .expect(200);
```

Comment:
should we add an expect (refreshtoken tobe null) in the database after revoke?

Reply:
That would be a good idea. But when I tried to add `.expect(refreshToken).toBeNull()`, I got an error saying "Property 'toBeNull' does not exist on type 'Test'"

Reply:
I added a separate test that checks that the user's refresh token in null